server/svix-server/src/cfg.rs

```diff
@@ -60,7 +77,7 @@ retry_schedule = "5,300,1800,7200,18000,36000,36000"
 # The DSN for redis (can be left empty if not using redis)
 # redis_dsn = "redis://redis:6379"
 
-# What kind of message queue to use. Supported: memory, redis (must have redis_dsn configured).
+# What kind of message queue to use. Supported: memory, redis (must have redis_dsn configured), rediscluster (add multiple hosts to redis_dsn separated by commas).
```

Comment:
I still feel like we should support multiple hostnames. It's a trivial change, and perhaps some users would want to specify multiple in case a server is down.

Reply:
That's fair. Though please clarify the doc then to clarify that you should only pass the main one under normal circumstances and only pass multiple if you want.

Though to be honest, since it's such a trivial change, I'd rather we waited until a requirement comes from an actual customer before we do it. What do you think?

Reply:
That's fine. I do want to clarify the request here, though. Currently there are 2 configuration changes in this PR:
1. Change `redis_dsn` parameter to support multiple hosts
2. Add optional `cache_redis_dsn` and `queue_redis_dsn`, which also support multiple hosts.

Do you want to leave in support for 2) and just remove support for multiple hosts, or or remove everything and only support `redis_dsn` with a single host for now?

Reply:
I'd say: remove everything and only support redis_dsn with a single host.

Follow up PR should allow overriding just one of the redis's, but don't think it's related to this PR.